### PR TITLE
add a basic CMakeLists.txt

### PR DESCRIPTION
**Pull request by a-teammate**
since SDL2 image is a really small project i think just adding our own CMakeLists.txt is okay. (Thanks to your already done conanfile.py it was easy).

This is working for me, tested in vs only though.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+# Project Name and start working in project scope (some things just work until now, some just work after this command)
+project(SDL2_image C)
+
+# Add conan dependencies
+include(../conanbuildinfo.cmake) # TODO This is actually ugly isnt it? Since it only works inside a "require", not using a normal "mkdir build && cd build && conan install .. && cmake .."
```
Comment (lasote) [inline CMakeLists.txt:7]: Yes, you can replace it with:

```
include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

**Comment by lasote** [comment]
It looks VERY good. I have to check why is failing in linux/osx. Thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (a-teammate) [reply CMakeLists.txt:7]: I tried before, but it wasn't working then for me anymore, not finding the include.. weird

**Comment by lasote** [comment]
I think the CMakeLists is not working well for linux/osx because the "configure" is really doing some things. With the CMakelists.txt it builds ok but then it can't find the png, jpeg libraries in runtime. Maybe some definition is missing. I think we could keep your fantastic CMakeLists.txt for windows and recover the linux/osx old system. What do you think?

**Comment by a-teammate** [comment]
Sorry, I ran the wrong test suite..
I'll fix some remaining errors today eve, e.g. I forgot the LOAD_*\* defines

**Comment by lasote** [comment]
The CI examples are crashing with "Segmentation fault (core dumped)". Something is not going well in linux with the CMake yet. https://travis-ci.org/lasote/conan-sdl2_image/jobs/174782553

**Comment by a-teammate** [comment]
Hm yeah and OSx also seems to fail..
Shit, since I can't debug this, I think your point of just merging the 2 working parts we have together seems like the best solution.

**Comment by lasote** [comment]
Hi, I've merged your changes keeping the autotools for linux and osx. Actually osx keeps failing with shared sdl2_image, I'll keep investigating.

Thanks!

**Comment by a-teammate** [comment]
:+1: Nice, thank you!
(Btw: I think you accidentally merged 4bc15c6349bea63e37bc1eb3b52d9fbc48f122b8 (changed SDL2/2.0.4@lasote/stable -> SDL2/2.0.5@a_teammate/testing). I suggest changing it to SDL2/2.0.5@lasote/stable now since that one got fixed too)

**Pull request merged**